client/src/components/Video.js

```diff
@@ -0,0 +1,112 @@
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faThumbsUp, faThumbsDown } from "@fortawesome/free-solid-svg-icons";
+
+const Video = ({
+  id,
+  title,
+  url,
+  ratingUp,
+  ratingDown,
+  getAllVideos,
+  getDeleteMessage,
+}) => {
+  const videoId = url.split("v=")[1];
+
+  const deleteHandler = async () => {
+    try {
+      const response = await fetch(`${process.env.REACT_APP_SERVERURL}/${id}`, {
+        method: "DELETE",
+      });
+      if (response.status !== 200) {
+        throw new Error("Something went wrong!");
+      }
+      const data = await response.json();
+      getDeleteMessage(data.message);
```

Comment:
It's not super clear from the name what `getDeleteMessage` does - could you think of a more clear name to show what this means?